Content.Client/Lathe/LatheSystem.cs

```diff
@@ -24,7 +26,8 @@ protected override void OnAppearanceChange(EntityUid uid, LatheVisualsComponent
                     sprite.LayerSetAnimationTime(LatheVisualLayers.IsRunning, 0f);
                     sprite.LayerSetState(LatheVisualLayers.IsRunning, state);
                 }
-                if (args.Component.TryGetData(LatheVisuals.IsInserting, out bool isInserting))
+                if (args.Component.TryGetData(LatheVisuals.IsInserting, out bool isInserting)
+                    && sprite.LayerMapTryGet(LatheVisuals.IsInserting, out _))
```

Comment:
Instead of `out _` this should just be using the out value when doing the layer set.